dev/src/convert.ts

```diff
@@ -44,19 +44,20 @@ const validate = createValidator();
  * @return {{nanos,seconds}|undefined} The value as expected by Protobuf JS or
  * undefined if no input was provided.
  */
-function convertTimestamp(timestampValue, argumentName) {
-  let timestampProto = undefined;
+function convertTimestamp(timestampValue, argumentName?) {
+  // tslint:disable-next-line:no-any
+  let timestampProto: any = undefined;
```

Comment:
You should be able to import the type definitions for this from https://github.com/googleapis/nodejs-firestore/blob/master/dev/protos/firestore_proto_api.d.ts